apps/rehabstod/src/utils/fake/fakePatient.ts

```diff
@@ -0,0 +1,11 @@
+import { faker, fakerFromSchema } from '@frontend/fake'
+import { GenerateMockOptions } from '@anatine/zod-mock'
+import { patientSchema, patientSjukfallSchema } from '../../schemas/patientSchema'
+
+const fakePatientSjukfall = fakerFromSchema(patientSjukfallSchema)
+
+export const fakePatient = fakerFromSchema(patientSchema, {
+  sjukfallList: Array.from({ length: faker.datatype.number({ min: 1, max: 5 }) }, fakePatientSjukfall),
+  srsError: faker.datatype.boolean(),
+  kompletteringInfoError: faker.datatype.boolean(),
+} as Omit<GenerateMockOptions, 'sjfMetaData'>)
```

Comment:
We can use `fakerFromSchemaFactory` to wrap our faker in another layer, this avoids the typescript issue.

```typescript
export const fakePatient = fakerFromSchemaFactory(patientSchema, {
  sjukfallList: Array.from({ length: faker.datatype.number({ min: 1, max: 5 }) }, fakePatientSjukfall),
  srsError: faker.datatype.boolean(),
  kompletteringInfoError: faker.datatype.boolean(),
})
```